man/systemd.netdev.xml

```diff
@@ -731,7 +731,8 @@
       <varlistentry>
         <term><varname>TTL=</varname></term>
         <listitem>
-          <para>Specifies the TTL value to use in outgoing packets. Ranges [1-255].</para>
+          <para>Specifies the TTL value to use in outgoing packets. Takes a number in the range 0-255.
+           When unset, the kernel's default will be used. </para>
```

Comment:
`0` means unset, so I do not think it is not necessary to include 0 in the range. That is, I prefer `1-255`.

Reply:
Changed a bit but updated what exactly means by 0 or unset.